### ISSUE TITLE
Standardize test messages to use the permissible constraints (i.e. the thing being asserted)

### ISSUE THREAD

**Issue by ccaprani** [pymc-devs/pymc#6158]
Following on from the discussion here https://github.com/pymc-devs/pymc-experimental/pull/84#discussion_r981865168 with @ricardoV94, in many places the test message is written in the opposite direction to the check it looks like, e.g. [Gumbel](https://github.com/pymc-devs/pymc/blob/b12c189896e260cb743d31a47a5d4680f2358216/pymc/distributions/continuous.py#L3320), [Logistic](https://github.com/pymc-devs/pymc/blob/b12c189896e260cb743d31a47a5d4680f2358216/pymc/distributions/continuous.py#L3545), [Normal](https://github.com/pymc-devs/pymc/blob/b12c189896e260cb743d31a47a5d4680f2358216/pymc/distributions/continuous.py#L584), [HalfNormal](https://github.com/pymc-devs/pymc/blob/b12c189896e260cb743d31a47a5d4680f2358216/pymc/distributions/continuous.py#L909), [Beta](https://github.com/pymc-devs/pymc/blob/b12c189896e260cb743d31a47a5d4680f2358216/pymc/distributions/continuous.py#L1287); e.g.

```python
check_parameters(res, 0 < beta, msg="beta > 0")
```

Others are written in the same direction, e.g. [TruncatedNormal](https://github.com/pymc-devs/pymc/blob/b12c189896e260cb743d31a47a5d4680f2358216/pymc/distributions/continuous.py#L793), [Wald](https://github.com/pymc-devs/pymc/blob/b12c189896e260cb743d31a47a5d4680f2358216/pymc/distributions/continuous.py#L1095), [Kumaraswamy](https://github.com/pymc-devs/pymc/blob/b12c189896e260cb743d31a47a5d4680f2358216/pymc/distributions/continuous.py#L1389) and so on.

```python
logp = check_parameters(
                logp,
                at.le(lower, upper),
                msg="lower_bound <= upper_bound",
            )
```
Messages should be standardized to use the permissable constraints; i.e. written in the same direction.

**Comment by ricardoV94** [comment]
That first example is fine, the expression is the same just written the other way around. The problem is if it has the opposite meaning

**Comment by ricardoV94** [comment]
Closed via #6272
